### PR TITLE
[WebAPI] Fix missing version information

### PR DESCRIPTION
**Pull request by rwalczyna**
Version information was missing for some of the methods:
- 'Since' tags missing

Also there was some grammar fixes in mediacontroller and humanactivitymonitor modules:
- fixed some descriptions and language mistakes
- fixed some 'description' tags with merging into one

### CODE REVIEW

**Comment by djuszczakk** [comment]
Changes in files mediacontroller.html for each profile were added automatically. Main pull request for these files is https://github.com/Samsung/tizen-docs/pull/1001

**Comment by pkosko** [comment]
Changes in humanactivitymonitor.html are basically from PR #997. Purpose of this PR is only adding missing information about 'since' and few language/description issues.

File: docs/application/web/api/6.0/device_api/wearable/tizen/mediacontroller.html [hunk 1]
```diff
@@ -3750,38 +3950,1475 @@ <h4>Attributes</h4>
...
+<dt class="method" id="MediaControllerServerInfoPlaybackInfo::sendRepeatState">
+<a class="backward-compatibility-anchor" name="::MediaController::MediaControllerServerInfoPlaybackInfo::sendRepeatState"></a><code><b><span class="methodName">sendRepeatState</span></b></code>
+</dt>
+<dd>
+<div class="brief">
+ Sends request to change the repeat state of a media controller server.
+            </div>
+<div class="synopsis"><pre class="signature prettyprint">void sendRepeatState(<a href="#MediaControllerRepeatState">MediaControllerRepeatState</a> state, optional <a href="#MediaControllerSendCommandSuccessCallback">MediaControllerSendCommandSuccessCallback</a>? replyCallback);</pre></div>
+<p><span class="version">Since: </span>
+ 5.5
```
Comment (mijin85cho) [inline docs/application/web/api/6.0/device_api/wearable/tizen/mediacontroller.html]: Is the "Since version" really 5.5?

**Comment by rwalczyna** [comment]
@mijin85cho, you are right, it should be 6.0. Fixed.

**Comment by mijin85cho** [comment]
Thanks for the update, @rwalczyna.
@pkosko Would you please check the latest files again and approve it if you see no more issues?

**Pull request merged**